p2p/server.py

```diff
@@ -104,13 +117,18 @@ def register_peer(self, peer: ETHPeer) -> None:
             auth_ack_ciphertext=auth_ack_ciphertext
         )
 
-        # Store peer creation data in incoming_connections[]
-        # TODO register peer in PeerPool
+        # Create and register peer in peer_pool
         peer = (aes_secret, mac_secret, initiator_nonce, ephem_pubkey)
+        eth_peer = ETHPeer(
+                remote=initiator_remote, privkey=None, reader=None,
+                writer=None, aes_secret=aes_secret, mac_secret=mac_secret,
```

Comment:
Is there a reason `reader` and `writer` are set to `None`?  Seems like the local `reader` and `writer` instances should be used here.